jenkins/Jenkinsfile

```diff
@@ -0,0 +1,246 @@
+def configuration = [vaultUrl: "${VAULT_URL}",  vaultCredentialId: "vault-app-role", engineVersion: 2]
+
+def secrets = [
+  [path: 'secret/jenkins/cloudify', engineVersion: 2, secretValues: [
+    [envVar: 'OS_USERNAME', vaultKey: 'username'],
+    [envVar: 'OS_TENANT', vaultKey: 'tenant'],
+    [envVar: 'OS_IP', vaultKey: 'ip'],
+    [envVar: 'OS_PASSWORD', vaultKey: 'password']]],
+]
+
+@Library('pipeline-shared-library') _
+
+pipeline {
+  agent {
+    kubernetes {
+      label "agent-${env.BUILD_NUMBER}"
```

Comment:
let's rename it to cloudify-agent-, we have too many agents

Reply:
ok, np

Reply:
done.